tests/IntegrationTest.php

```diff
@@ -34,6 +48,20 @@ public function testTwigIntegration()
             '<script src="/build/file1.js"></script>',
             $html2
         );
+        $this->assertContains(
+            '<script src="/build/other4.js"></script>',
+            $html2
+        );
+        // styles3.css is not repeated
+        $this->assertNotContains(
+            '<link rel="stylesheet" href="/build/styles3.css">',
+            $html2
+        );
+        // styles4.css is not repeated
+        $this->assertNotContains(
+            '<link rel="stylesheet" href="/build/styles4.css">',
+            $html2
+        );
```

Comment:
Hmm, could we split this into a separate test instead of adding to this test? It makes it a bit harder to understand.